src/modules/email/interfaces/email.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface SendData {
+  to: String,
+  templateId: String,
```

Comment:
Consider `enum` or `type`

Reply:
That's a good point.  I've switched it tp an `enum` now.